src/server.js

```diff
@@ -5,9 +5,9 @@ const pug = require('pug');
 const http = require('http');
 const port = 8000;
 
-app.set('views', '../views');
```

Comment:
This one dot/two dots issue is very weird! I can't really think of any fixes though. Do we know if it's just Michael's machine that needs two or are other people finding that too?